compositor/sessionmanager/sessionmanager.cpp

```diff
@@ -320,9 +285,6 @@ void SessionManager::requestHybridSleep()
     Q_EMIT hybridSleepRequested();
 }
 
-void SessionManager::cancelShutdownRequest()
-{
-    Q_EMIT shutdownRequestCanceled();
-}
+void SessionManager::cancelShutdownRequest() { Q_EMIT shutdownRequestCanceled(); }
 
-#include "moc_sessionmanager.cpp"
+#include "sessionmanager.moc"
```

Comment:
It works fine without including the `moc_*.cpp` files. However, it's necessary to include the `*.moc` here because `Q_OBJECT` is used in the C++ file. See http://stackoverflow.com/q/34928933.